ajax/libs/ityped

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "ityped",
+	"filename": "dist/index.min.js",
```

Comment:
no need to use dist/ in filename. You already fetch in inside dist with auto update config